scripts/geopmpy/io.py

```diff
@@ -795,7 +791,7 @@ def get_field(self, raw_data, key, units=''):
 
 class RawReportCollection(object):
     '''
-    Used to group together a collection of related RawReports.
+    Used to group together a collection of related ``RawReport``\ s.
```

Comment:
Similar to another comment about `:py:func:` you could use `:py:class:` here so that readers of the documentation can just click on `RawReport` to get to its documentation.